composer.json

```diff
@@ -94,6 +94,13 @@
         "su-sws/jumpstart_ui": "dev-8.x-1.x",
         "su-sws/stanford_text_editor": "dev-8.x-1.x"
     },
+    "require-dev": {
+        "drupal/config_pages": "^2.7",
+        "drupal/field_encrypt": "^2.0@alpha",
+        "drupal/real_aes": "^2.3",
+        "sainsburys/guzzle-oauth2-plugin": "^3.0",
+        "su-sws/stanford_migrate": "dev-D8CORE-1736"
+    },
```

Comment:
Why require-dev? If you want to enable the stanford_person_importer module you will need these. I'd argue for this module + children modules to be functional these are required.

Reply:
I did this because we don't need to require them if the person don't want to enable the importer. the parent module is still functional.

Reply:
I'd prefer the opposite. I'd rather have everything required to enable all modules by default even if they are not enabled.

Reply:
Me. I doesn't really matter enough to me. I see benefits both ways. `:meh:`